treeherder/client/thclient/client.py

```diff
@@ -701,6 +708,40 @@ def _post_json(self, project, endpoint, oauth_key, oauth_secret, jsondata,
                              timeout=timeout)
         resp.raise_for_status()
 
+    def _get_json(self, project, endpoint, timeout=None):
```

Comment:
as per the comments below, you should add a `params` parameter that you can pass to requests.get to specify a querystring